RadixWallet/Features/DappInteractionFeature/Children/DappOriginVerification/DappInteractionOriginVerification +View.swift

```diff
@@ -39,7 +39,7 @@ extension DappInteractionOriginVerification {
 					.textStyle(.sheetTitle)
 					.padding(.bottom, .large2)
 
-				Text(L10n.MobileConnect.linkSubtitle(store.dAppMetadata.name))
+				Text(markdown: L10n.MobileConnect.linkSubtitle(store.dAppMetadata.name), italicsColor: .app.gray1)
```

Comment:
Maybe we should rename the init to be
```Swift
public init(markdown: String, emphasizedColor: Color)
```

because it actually applies for italics, bold and whatever else is empashized.